### ISSUE TITLE
OWASP Dependency Check version 6 (dependency-check-build-task@6) is dependent on a Node version (16) that is end-of-life

### ISSUE THREAD

**Issue by raphaelmoreira** [dependency-check/DependencyCheck#7927]
Hello.

Today we start to receive this warning:

```
[warning]Task 'OWASP Dependency Check' version 6 (dependency-check-build-task@6) is dependent on a Node version (16) that is end-of-life. 
Contact the extension owner for an updated version of the task. 
Task maintainers should review Node upgrade guidance: https://aka.ms/node-runner-guidance
```

This tool will be updated on [DevOps Marketplace](https://marketplace.visualstudio.com/items?itemName=dependency-check.dependencycheck&ssr=false#overview)?

**Comment by jeremylong** [comment]
No one who works on the core dependency-check project is involved with the azure pipeline.

**Comment by ThomasAy** [comment]
@raphaelmoreira you might want to take a look at this issue : https://github.com/dependency-check/azuredevops/issues/179
